src/aiq/memory/models.py

```diff
@@ -13,6 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# yapf: disable
```

Comment:
Can you only disable if for the schema section and re-enable it after?

Reply:
Ack